sample-data/sdo-dashboard-single-pipeline.json

```diff
@@ -459,7 +459,7 @@
                     ]
                 ],
                 "view": "timeSeries",
-                "title": "flaky-service-pipeline MTTR",
+                "title": "app1-flaky-service-pipeline MTTR",
```

Comment:
You will need more test cases where app1 is in the middle of the name and when app1 is at the end

Reply:
Added a few more,  I ll request a re-review once i am done with that.